### DISCUSSION TITLE
Creality BL support?

### DISCUSSION THREAD

**Discussion by achillez** [mriscoc/Ender3V2S1#185] [Q&A]
Thinking to upgrade my Ender 3 v2. Creality has a decent BL unit. is this compatible with the firmware? I looked through the configs: 
https://github.com/mriscoc/Special_Configurations

But didn't see the CR BL as an option.

**Comment by achillez** [comment]
Doh... searched a bit more and saw CR is available. BLTouch I assume. 

Maybe related Q, is this the best BL to get for the $?

**Comment by mriscoc** [comment]
Some people have problems with the CRTouch, HS mode and large mesh leveling. I have a 3DTouch v3.1 from Trianglelabs and never had any problem.
